doc/tutorial/quick-start.md

```diff
@@ -27,11 +27,11 @@
 
 ### Run the image build
 
-- Run 
+- Choose the OpenStack <cloud-name> from the clouds.yaml file, and set the desired image name as <image-name>.
```

Comment:
I think that `<cloud-name>` and `<image-name>` need to be formatted as code, otherwise they don't show up in the "rich diff" view.